carmajava/mock_drivers/src/main/java/gov/dot/fhwa/saxton/carma/mock_drivers/MockDriverNode.java

```diff
@@ -44,6 +44,7 @@
     final Log log = connectedNode.getLog();
     final ParameterTree params = connectedNode.getParameterTree();
     final IMockDriver simulatedDriver;
+    String switchValue = params.getString("~/simulated_driver");
```

Comment:
If you are going to make this variable you should also use it in the switch(). Right now there are duplicate param calls